### ISSUE TITLE
Display doesn't shutdown (windows 10)

### ISSUE THREAD

**Issue by raf47** [cDima/Aerial#117]
Hi, 
The screensaver works perfectly (good job by the way), except that the monitor never shuts down. Even though it is set up to shutdown after 15 minutes, the screensaver just keeps running until the computer goes to sleep mode after 2 hours. I tried toggling the screensaver options, without any luck. 
Any ideas ? thanks for your help.

**Comment by chrisrlong** [comment]
I noticed this too, however currently sometimes it works, sometimes the screens go straight to sleep.

I changed the screen saver time to 3 mins and the power to 15mins. (I used to have it on 'never' - but i had complaints that it looked like the office lights were on over night.)

**Comment by chrisrlong** [comment]
I have observed a bit more closely:

Scenario 1: computer is unlocked and logged in, screensaver comes on as expected, but does not allow power settings to power down the screens.

Scenario 2: computer is locked (start + l), screen saver does not start, the screens go straight into standby.

**Comment by BakaNode** [comment]
This is also an issue on both Windows machines I've installed it on. Both machines start the screensaver perfectly fine but the display never goes to sleep.

This is not an issue on the Mac version this work was developed from. Maybe something in power management got lost in translation?

**Comment by mahtar** [comment]
Same issue here. Temporary workaround below but it affects all the other applications that prevent automatic sleep as well (video players, browser when playing video, etc.)

1. Open Local Group Policy Editor (GPedit.msc).
2. Go to Computer Configuration -> Administrative Templates -> System -> Power Management -> Sleep Settings.
3. Set the following two settings to Disabled:
Allow Applications to Prevent Automatic Sleep (On Battery) 
Allow Applications to Prevent Automatic Sleep (Plugged In)

**Comment by raf47** [comment]
Thanks for the info, howver that didn't work for me, I don't seem to have these options in the group Policy editor...

**Comment by ninjit** [comment]
Experience the same issue here, and I believe @mahtar 's workaround points to the problem.
I think Aerial is implemented as a video overlay, and by default playing videos prevents screen sleep.

**Comment by stephenjmcmahon** [comment]
same issue - verified all my settings and its set to turn off at 45 (and did so prior to installing)

**Comment by BakaNode** [comment]
I don’t think this will ever be fixed. The author seems to have abandoned their work. I wish someone could take it over and keep it running like the Mac version.

**Comment by koperak** [comment]
I have tested [this](https://github.com/thoemmi/AerialForWindows) aerial screensaver and works as expected - just do monitor off.

**Comment by Correz** [comment]
I realise that this thread is a little old; however at least on my Windows 10 machine this application is using "Windows Media Player" to play .mov files within Aerial for Windows and WMP is set by default to disable screensaver / power off while it is playing.  You can disable this behaviour in WMP via Organise > Options > Player > Check 'Allow screen saver during playback'. And Voila! The screen will now power down based on the interval specified in Windows power settings even when aerial is playing.

**Comment by lalatte** [comment]
Interesting answer, I will try it ! thanks

**Comment by lalatte** [comment]
It works perfectly indeed. Well done !

**Comment by gitanO0** [comment]
> I realize that this thread is a little old; however, at least on my Windows 10 machine this application is using "Windows Media Player" to play .mov files within Aerial for Windows and WMP is set by default to disable screensaver / power off while it is playing. You can disable this behavior in WMP via Organise > Options > Player > Check 'Allow screen saver during playback'. And Voila! The screen will now power down based on the interval specified in Windows power settings even when aerial is playing.

What a great solution.  You rock.
